conf/linux64-defconfig

```diff
@@ -2967,7 +2968,9 @@ CONFIG_MESSAGE_LOGLEVEL_DEFAULT=4
 #
 # Compile-time checks and compiler options
 #
-# CONFIG_DEBUG_INFO is not set
+CONFIG_DEBUG_INFO=y
+CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
```

Comment:
This should be actually added by `cmake`'s additional flag `-DDebug`. (FYI, I created #278, so you don't need to fix it in this PR)

Reply:
Same situation as above -- did not mean to push this change.